crypto/pem/pem_lib.c

```diff
@@ -610,6 +610,8 @@ int PEM_write_bio(BIO *bp, const char *name, const char *header,
     unsigned char *buf = NULL;
     EVP_ENCODE_CTX *ctx = EVP_ENCODE_CTX_new();
     int reason = ERR_R_BUF_LIB;
+    int retval = 0;
+    int bufsize = 0;
```

Comment:
Why do we need this? It only ever seems to be written to and never read.

Reply:
At first I was thinking the buffer got resized.  I will remove this.